css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+  font-family: "DM Sans", sans-serif;
+}
+
+.container {
+  max-width: 2000px;
+  margin: auto;
+}
+
+.background-color {
+  background-color: #fff;
+}
+
+.row {
+  display: flex;
+}
+
+.padding-row {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-fifth {
+  flex-basis: calc(100% / 4);
```

Comment:
This looks like it should be `column-fourth` based on this measurement. I imagine you meant `20%`